kythe/rust/indexer/src/error.rs

```diff
@@ -41,6 +41,11 @@ quick_error! {
             from()
             display("Failed to parse Protobuf: {}", err)
         }
+        /// There was an issue serializing a type to JSON
+        JsonSerializationError(err: serde_json::Error) {
+            from()
```

Comment:
Where is this used?

Reply:
When the requests are generated, the serde_json serialization returns a `serde_json::Result` which may include a `serde_json::Error`. This code converts any `serde_json::Error` into a `KytheError::JsonSerializationError`.